src/GafferSceneUIModule/RenderPassEditorBinding.cpp

```diff
@@ -346,6 +346,24 @@ class RenderPassPath : public Gaffer::Path
 			return m_scene.get();
 		}
 
+		const Gaffer::Context *inspectionContext( const IECore::Canceller *canceller ) const override
+		{
+			const auto renderPassName = runTimeCast<const IECore::StringData>( property( g_renderPassNamePropertyName, canceller ) );
```

Comment:
> Ah, I see my assumption about inspectionContext() not needing cancellation was wrong. We do need cancellation for this because it computes the scene globals, correct?

Correct, we need cancellation as we'd be computing the scene globals here.

> I still wonder if it would be better to remove the responsibility of putting the canceller in the returned context...

Yeah, it does still feel a little bit odd putting the responsibility here. Might be worth seeing how this goes with another Path, such as the one needed for the SceneInspector revamp and then we could consider changing things up before the release of 1.5, or go with the other option we discussed of a public non-virtual method on Path that calls a non-public virtual `inspectionContext()` and ensures things are set up correctly on the way out...